modules/living_spaces_discussions/config/rewrite/group.role.telephone_counseling-f9845542f.yml

```diff
@@ -0,0 +1,5 @@
+permissions:
+  - 'create group_node:discussion_post entity'
+  - 'delete any group_node:discussion_post entity'
+  - 'update any group_node:discussion_post entity'
+  - 'view group_node:discussion_post entity'
```

Comment:
This role won't be updated during module install because it won't exist. in dip_counseling which is installed way after living_spaces